variables.tf

```diff
@@ -122,14 +118,18 @@ should use.
 
 This is specified as a list of objects:
 
-    storage_lifecycle_rules = [{
-      action = [{
+    storage_lifecycle_rules = [
+      {
         type = "Delete"
-      }]
-      condition = [{
-        num_newer_versions = 3
-      }]
-    }]
+        conditions = {
+          age = 60,
+          is_live = false
+          matches_storage_class = ["REGIONAL"]
```

Comment:
I know this is just a default, but we absolutely want to use multi_regional for the actual edition if possible.